### ISSUE TITLE
the Subject  can also be invalid UTF-8

### ISSUE THREAD

**Issue by judell** [turbot/steampipe-plugin-imap#2]
I'm working around it like so:

```
	if !utf8.ValidString(te.BodyText) {
		te.BodyText = "invalid UTF-8"
	}
	if !utf8.ValidString(env.HTML) {
		te.BodyHTML = "invalid UTF-8"
	}
	if ! utf8.ValidString(te.Subject) {
		te.Subject = "invalid UTF-8"
	}
```

**Comment by e-gineer** [comment]
I believe this is fixed in af099006f6026d40415901c0c07342a5f0aafdc5

**Comment by judell** [comment]
Yep, thanks.
